sdk/spring/azure-spring-boot-test-selenium-common/src/main/java/com/azure/test/aad/common/SeleniumITHelper.java

```diff
@@ -0,0 +1,115 @@
+        final String chromedriverMac = "chromedriver_mac64";
+        String osName = System.getProperty("os.name").toLowerCase();
+        Process process = null;
+        File dir;
+        try {
+            if (Pattern.matches("linux.*", osName)) {
+                dir = copyChromeDriverFile(strTmpPath, chromedriverLinux);
+                process = Runtime.getRuntime().exec("chmod +x " + chromedriverLinux, null, dir.getParentFile());
+                process.waitFor();
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, dir.getPath());
+            } else if (Pattern.matches("windows.*", osName)) {
+                dir = copyChromeDriverFile(strTmpPath, chromedriverWin32);
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, dir.getPath());
+            } else if (Pattern.matches("mac.*", osName)) {
+                dir = copyChromeDriverFile(strTmpPath, chromedriverMac);
+                process = Runtime.getRuntime().exec("chmod +x " + chromedriverMac, null, dir.getParentFile());
+                process.waitFor();
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, dir.getPath());
+            } else {
+                throw new IllegalStateException("Unrecognized osName. osName = " + System.getProperty("os.name"));
+            }
+        } catch (InterruptedException | IOException e) {
+            throw new RuntimeException(e);
+        } finally {
+            if (process != null) {
+                process.destroyForcibly();
+            }
+        }
+    }
+
+    protected void createDriver() {
+        if (driver == null) {
+            ChromeOptions options = new ChromeOptions();
+            options.addArguments("--headless");
+            options.addArguments("--incognito", "--no-sandbox", "--disable-dev-shm-usage");
+            driver = new ChromeDriver(options);
+            wait = new WebDriverWait(driver, 10);
+        }
+    }
+
+    protected void createAppRunner(Class<?> appClass, Map<String, String> properties) {
+        app = new AppRunner(appClass);
+        properties.forEach(app::property);
+        app.start();
+    }
+
+    private static File copyChromeDriverFile(String strTmpPath, String chromeDriverName)
+        throws IOException {
+        InputStream resourceAsStream = SeleniumITHelper.class.getClassLoader()
+            .getResourceAsStream("driver/" + chromeDriverName);
+        File dest = new File(strTmpPath + File.separator + uuid + File.separator + chromeDriverName);
+        FileUtils.copyInputStreamToFile(resourceAsStream, dest);
+        return dest;
+    }
+
+    private static void deleteChromeDriverFile() {
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            String strTmpPath = System.getProperty("java.io.tmpdir");
+            File targetFile = new File(strTmpPath + File.separator + uuid);
+            try {
+                FileUtils.deleteDirectory(targetFile);
```

Comment:
The `java.io.File` already has a `deleteOnExit` method, we should leverage that. And apache commons also provides the `forceDeleteOnExit` method to get rid of a directory. Let's change these again.